src/commands/Misc/ship.ts

```diff
@@ -0,0 +1,131 @@
+import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { CanvasColors } from '@lib/types/constants/Constants';
+import { UserSettings } from '@lib/types/settings/UserSettings';
+import { KeyedMemberTag } from '@root/src/arguments/membername';
+import { socialFolder } from '@utils/constants';
+import { fetch, FetchResultTypes, getDisplayAvatar, loadImage } from '@utils/util';
+import { Image } from 'canvas';
+import { Canvas } from 'canvas-constructor';
+import { remove as removeConfusables } from 'confusables';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { join } from 'path';
+
+export default class extends SkyraCommand {
+
+	private removeSymbolsRegex = /(?:[~`!@#%^&*(){}[\];:"'<,.>?/\\|_+=-])+/g;
```

Comment:
```suggestion
	private readonly kRemoveSymbolsRegex = /(?:[~`!@#%^&*(){}[\];:"'<,.>?/\\|_+=-])+/g;
```

And update all references